README.md

```diff
@@ -4,7 +4,7 @@ How to design and implement an API which can perform CRUD operations on a databa
 
```

Comment:
I know you mentioned the README is not finished. Here are a few tips of things to include when you do get chance to finish it (feel free to include other things you want too:
1) an explanation of what the project is and why you are doing it
2) what technologies are used, including dependencies and dev dependencies
3) how would someone download your repo and get setup to run `npm start` or `npm test``
4) a list of all the routes available and what data needs to be sent with them (e.g. for POST and PUT/PATCH requests to artist they would send in req.body name and genre as varchar and these can't be null